src/Ses/Input/SendEmailRequest.php

```diff
@@ -127,35 +127,44 @@ public function getReplyToAddresses(): array
     public function requestBody(): string
     {
         $payload = ['Action' => 'SendEmail', 'Version' => '2019-09-27'];
+        $indices = new \stdClass();
         if (null !== $v = $this->FromEmailAddress) {
             $payload['FromEmailAddress'] = $v;
         }
 
         if (null !== $this->Destination) {
-            (static function (Destination $input) use (&$payload) {
-                (static function (array $input) use (&$payload) {
+            (static function (Destination $input) use (&$payload, $indices) {
+                (static function (array $input) use (&$payload, $indices) {
+                    $indices->kbe21867 = -1;
                     foreach ($input as $value) {
-                        $payload['Destination']['ToAddresses']['member'][] = $value;
```

Comment:
I was wrong. The official sdk does not use `member` :-(.

But I don't have only Curl sample to assert it :-(